starters/oasp4j-batch-starter/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.oasp.java.dev</groupId>
+    <artifactId>oasp4j-starters</artifactId>
+    <version>dev-SNAPSHOT</version>
+  </parent>
+  <groupId>io.oasp.java.starters</groupId>
+  <artifactId>oasp4j-batch-starter</artifactId>
```

Comment:
Please follow spring conventions and put "starter" keyword before the topic like "batch". See also existing starter modules (and yammer discussions about it):
https://github.com/oasp/oasp4j/tree/develop/modules